File: Study/Date/MainActivity.java
import androidx.appcompat.app.AppCompatActivity;

import java.text.SimpleDateFormat;
import java.util.Date;

public class MainActivity extends AppCompatActivity {
  
    private long mNow;
    private Date mDate;
    private SimpleDateFormat mFormat = new SimpleDateFormat("MM/dd"); //표현하고자 하는 날짜 형식 지정.
    private TextView date;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_add);
      
        date = view.findViewById(R.id.date);
        date.setText(getTime());
    }
  
  //시스템으로부터 시간을 받아와 지정한 형식대로 날짜 데이터를 반환하는 메소드 직접 작성.
  private String getTime() { 
        mNow = System.currentTimeMillis();
        mDate= new Date(mNow);
        return mFormat.format(mDate);
    }
}

File: Study(Kotlin)/onClick/MainActivity.kt
package edu.ourincheon.myapplication

import android.os.Bundle
import android.util.TypedValue
import android.view.View
import android.widget.EditText
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.constraintlayout.widget.ConstraintLayout
import androidx.core.content.ContextCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val bgId: ConstraintLayout = findViewById(R.id.screen_background)
        //val bgColor = resources.getColor(R.color.color_background);
        val bgColor = ContextCompat.getColor( this, R.color.color_background)
        bgId.setBackgroundColor(bgColor)
    }

    fun onBtnClicked(view: View) {

        val dollars: EditText = findViewById(R.id.dollarEditText)
        var tv: TextView = findViewById(R.id.textView)

        var textColor = ContextCompat.getColor(this, R.color.color_textview)
        val textSize = resources.getDimension(R.dimen.text_size)

        if(dollars.text.isNotEmpty()) {
            tv.setTextColor(textColor);
            tv.setTextSize(TypedValue.COMPLEX_UNIT_DIP,textSize)

            var dollorValue = dollars.text.toString().toInt()
            var wonValue = dollorValue * 1136;
            tv.text = wonValue.toString() +  " 원"
        }else{
            tv.text = "No value"
        }

        //tv.text = "안녕, 안드로이드!" // 코틀린의 장점->속성을 직접 지정가능.
        //tv.setText("안녕, 안드로이드!") Java
    }


}
